CONTRIBUTING.md

```diff
@@ -148,4 +148,4 @@ Contributing to `loopback-workspace` is easy. In a few simple steps:
 ```
 
 [Google C++ Style Guide]: https://google.github.io/styleguide/cppguide.html
-[Google Javascript Style Guide]: https://google.github.io/styleguide/javascriptguide.xml
\ No newline at end of file
+[Google Javascript Style Guide]: https://google.github.io/styleguide/javascriptguide.xml
```

Comment:
We should point to our own style guide: http://loopback.io/doc/en/contrib/style-guide.html